src/toolbars/ToolBarButtons.cpp

```diff
@@ -0,0 +1,125 @@
+#include <utility>
```

Comment:
This new source file should have comments at top about its purpos

Reply:
I think we should stop that practice and stick just to SPDX license identifiers